packages/contracts/contracts/registry/InterfaceBasedRegistry.sol

```diff
@@ -64,6 +64,7 @@ abstract contract InterfaceBasedRegistry is DaoAuthorizableUpgradeable, UUPSUpgr
 
         if (entries[_registrant]) revert ContractAlreadyRegistered({registrant: _registrant});
 
+        // TODO: if we keep using this contract, instead of AdaptiveERC165, use ERC165Checker from OZ.
```

Comment:
Let's do it now. Otherwise, we would not be able to delete AdaptiveERC165 if we realize that we don't need it.

Reply:
done